src/main/java/org/opendatakit/suitcase/net/DownloadTask.java

```diff
@@ -72,8 +73,10 @@ protected Void doInBackground() throws IOException, JSONException {
     publish(new ProgressBarStatus(0, PROCESSING_ROW, false));
     RFC4180CsvWriter csvWriter = null;
     try {
-      csvWriter = new RFC4180CsvWriter(new FileWriter(
-          FileUtils.getCSVPath(cloudEndpointInfo, csv.getTableId(), csvConfig, savePath).toString()
+      csvWriter = new RFC4180CsvWriter(Files.newBufferedWriter(
```

Comment:
Wouldn't you want to use the Files.newBufferedWriter that allows you to explicitly set the character set?

https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html#newBufferedWriter-java.nio.file.Path-java.nio.charset.Charset-java.nio.file.OpenOption...-

Reply:
Fixed